gpytorch/kernels/additive_structure_kernel.py

```diff
@@ -49,7 +49,7 @@ def forward(self, x1, x2, batch_dims=None, **params):
             evaluate = True
             res = NonLazyTensor(res)
 
-        res = res.sum(-3).unsqueeze(0)
+        res = res.sum(0)
```

Comment:
`res = res.sum(-3)` - so that this works for batch mode

Reply:
Relative indexing won't work now that we support multibatch. I made batch_dims always swap the `d dimension to be the first batch dimensions, hence the sum(0).